src/main/kotlin/the/coding/force/exploring_kotlin_coroutines/configuration/CoroutineConfig.kt

```diff
@@ -0,0 +1,14 @@
+package the.coding.force.exploring_kotlin_coroutines.configuration
+
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.reactor.asCoroutineDispatcher
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import reactor.core.scheduler.Schedulers
+
+@Configuration
+class CoroutineConfig {
+
+    @Bean
+    fun ioDispatcher(): CoroutineDispatcher = Schedulers.boundedElastic().asCoroutineDispatcher()
```

Comment:
Pode usar o Dispatcher de forma nativa do Kotlin

```suggestion
    fimport kotlinx.coroutines.CoroutineDispatcher
import kotlinx.coroutines.Dispatchers
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class CoroutineConfig {

    @Bean
    fun ioDispatcher(): CoroutineDispatcher = Dispatchers.IO

}
```